features/magazine-medium-digests.feature

```diff
@@ -0,0 +1,10 @@
+@magazine
+Feature: Magazine 'eLife digests' list
+
+  Rules:
+  - Latest 3 articles that appear on https://medium.com/@elife
+
+  Scenario: List shows latest digests
```

Comment:
maybe an explicit `Given that Science Digests are not enabled` could make it unambiguous

Reply:
We will actually want isolation of the environment variable too.